### DISCUSSION TITLE
New Bie

### DISCUSSION THREAD

**Discussion by DAVID-ALANYO** [microsoft/IoT-For-Beginners#379] [Q&A]
i am  a anew person in IOT how can i start

**Comment by Rohan03march** [comment]
On Sat, 28 May 2022 at 5:45 PM, Deyvid ***@***.***> wrote:

> i am a anew person in IOT how can i start
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/IoT-For-Beginners/discussions/379>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AU2E2KSVL6QT2DVX7YXIVELVMIE5VANCNFSM5XGSFPLQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
Yes as per the issue I will be responding to the quarries shortly
As you can see IOT is a concept that it will take atleast  1 week of time
to complete the program

I would like to attend the summit the is held on 1st of June
Thank you .

**Reply by DAVID-ALANYO** [reply]
ok
thank you very much
